src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -883,11 +883,27 @@ void FancyZones::MoveSizeStartInternal(HWND window, HMONITOR monitor, POINT cons
     {
         m_zoneWindowMoveSize = iter->second;
         m_zoneWindowMoveSize->MoveSizeEnter(window, m_dragEnabled);
+        if (m_settings->GetSettings().showZonesOnAllMonitors)
+        {
+            for (auto [keyMonitor, zoneWindow] : m_zoneWindowMap)
+            {
+                if (zoneWindow && zoneWindow != iter->second)

```

Comment:
`zoneWindow != iter->second`
why do we need to skip `m_zoneWindowMoveSize`?

Reply:
I think we don't _need_ to, I just didn't want to call that function twice.

Reply:
let's name the condition something like `const bool moveSizeEnterCalled = zoneWindow != m_zoneWindowMoveSiz;`, so we don't have to look up the reason for the condition each time.

Reply:
Isn't it apparent what I'm doing here? I'm treating `iter->second` (e.g. `m_zoneWindowMoveSize`) in one way, and every _other_ `zoneWindow` in another way.

Reply:
I can see the logic, but there's no intent visible for why you're doing that, so anyone reading this code in the future will have to go through all the functions called above that line, understand what they do, take a _guess_ that the _only_ reason you've intended to skip it is because of the double call to `ShowZoneWindow`. Imagine that the future code will change and you'll also have to track all the conditions leading to the `ShowZoneWindow` in `MoveSizeEnter` etc. Adding just a single line will eliminate the future possibility of spending time for everyone on the thing you already know.

Reply:
Can I write a comment explaining what you said instead?

Reply:
sure, the only reason I've proposed that is to minimize implicit context.

Reply:
In my opinion if its possible to add literals like
 const bool moveSizeEnterCalled = zoneWindow != m_zoneWindowMoveSiz;  
I far better than adding comments. Cause literals are tied to the code. Comments are not and can be outdated easyly

Reply:
Yes, but if some code changes, the variable's name will no longer represent the real situation. In this case a variable is as good as a comment.

Reply:
I can happen, but variable and function names should be updated when changing the code.
Also its guaranteed that when changing the code, variable name and its usage will be shown in pull request as reviewed code. Comments are not and there for can be easier overlooked.
It was established in previous review that if something can be explained by descrptive variable and fuction name it's prefered over comment